README.md

```diff
@@ -213,6 +213,78 @@ Request Sequence:
 ================================== 1 failed in 1.2 seconds =================================
 ```
 
+### Endpoint Specific Fixtures
+Let's say that we have another endpoint , `get_user_by_id`, that requires a different kind of `userID`. We can't use the existing `userID` fixture, since it generates the wrong type of ID. We can solve this by writing an endpoint specific fixture.
+
+
+```python
+# fixtures.py
+import fuzz_lightyear
+
+
+@fuzz_lightyear.register_factory('userID')
+def create_biz_user_id(businessID):
+    return businessID + 1
+
+
+@fuzz_lightyear.register_factory('userID', endpoint_ids=["get_user_by_id"])
+def create_user_id():
+    return 'foo'
+
+
+@fuzz_lightyear.register_factory('businessID')
+def create_business():
+    return 1
```

Comment:
We don't actually need this in this example, right?

Reply:
I think this is good to have since it explicitly shows another fixture registered under the same name.